@navikt/core/css/baseline/fonts.css

```diff
@@ -1,28 +1,31 @@
+/* New Name */
 @font-face {
-  font-family: "Source Sans Pro";
+  font-family: "Source Sans 3";
   font-style: italic;
-  font-weight: 400;
+  font-weight: 400 600;
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/CSS/@font-face/font-weight
```
font-weight: 400 600;
```
er ikke 1 til 1 med gammel implementasjon da dette åpner opp for at font-weight settes til alt mellom 400 og 600, så f.eks 500 vil være mulig.

Reply:
Ja, den fleksibiliteten var i utgangspunktet ønskelig for vår del, men ser at `400 600` ikke dekker behovet helt. Vi har brukt 650 for overskrifter i nye maler.